lib/internal/bootstrap_node.js

```diff
@@ -225,12 +225,27 @@
     global.setTimeout = timers.setTimeout;
   }
 
+  function setupConsoleAPI(addCommandLineAPIMethod) {
+    addCommandLineAPIMethod('require', function require(request) {
```

Comment:
Moved `setupConsoleAPI` below and renamed it to `setupInspectorAPI` (closer to `addCommandLineAPIMethod`). Also switched to accessing the method from process instead of passing it.